frontend/src/pages/OperationDetail.tsx

```diff
@@ -165,6 +185,31 @@ export const OperationDetail: React.FC = () => {
                     setCachedOperation(operationId, updatedOperation);
                 }}
             />
+
```

Comment:
Probably here I should display all the invoices linked to this specific operation

Reply:
Generally speaking: how the customer can understand which operation is associated to the invoice that he sees in the patient detail section?